projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2816,6 +2816,13 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'sanitize_callback' => 'sanitize_text_field',
 				'jp_group'          => 'wordads',
 			),
+			'wordads_cmp_enabled'                  => array(
+				'description'       => esc_html__( 'Enable A8C EU Consent Management Banner for WordAds', 'jetpack' ),
```

Comment:
I don't know if site owners will know what "A8C" stands for.

Reply:
Is this part user-facing? How can I view it? 
The settings option itself says `Enable GDPR Consent Banner`.

Reply:
It does not immediately appear, but since the string is translatable it will be offered to the community for translation.

Reply:
Got it, thanks! I have updated it for clarity.